<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your AI Teacher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Your AI Teacher</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToggleMode">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>☀️</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="mainContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelInstruction">
         <property name="text">
          <string>Nhập bài viết tiếng Anh cần đánh giá:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textInput">
         <property name="placeholderText">
          <string>Enter your text here...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelLevel">
           <property name="text">
            <string>Chọn trình độ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboLevel">
           <item>
            <property name="text">
             <string>Beginner</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Intermediate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Advanced</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEvaluate">
           <property name="text">
            <string>Đánh giá bài viết</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelResult">
         <property name="text">
          <string>Kết quả đánh giá:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Kết quả đánh giá sẽ hiển thị ở đây...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelQuestion">
         <property name="text">
          <string>Đặt câu hỏi thêm về bài viết:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="questionLayout">
         <item>
          <widget class="QLineEdit" name="lineQuestion">
           <property name="placeholderText">
            <string>Nhập câu hỏi của bạn...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAsk">
           <property name="text">
            <string>Hỏi</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>